modules/sara_r4/cellular_r4_api.c

```diff
@@ -2825,3 +2851,27 @@ CellularError_t Cellular_Init( CellularHandle_t * pCellularHandle,
 }
 
 /*-----------------------------------------------------------*/
+
+/* FreeRTOS Cellular Library API. */
+/* coverity[misra_c_2012_rule_8_7_violation] */
```

Comment:
What is this violation and What is the justification for this violation?

Reply:
This comment is moved from cellular_r4_wrapper.c together with Cellular_SocketSetSockOpt.